docs/source/federation/implementing.md

```diff
@@ -75,7 +77,14 @@ const typeDefs = gql`
     username: String
   }
 `;
+```
+
+The `@key` directive tells other services which fields to use to uniquely identify
```

Comment:
Perhaps useful to mention that this is a selection set on the object. For instance, key fields on User can be
```
@key( fields: "name
accounts {
  id
}")
```